### DISCUSSION TITLE
SSL certificates not generating

### DISCUSSION THREAD

**Discussion by JonQuayle** [vercel/community#3654] [Help]
### Summary

Hi, I have two websites hosted on Vercel. When visiting one I get a 'can't be found message' and another 'this connection is not private. After chatting to my domain registrar thinking it was something to do with that, they notified me that the SSL certificate's have expired.

When checking the domains settings in Vercel everything is configured correctly, but the 'generating SSL certificate' notice is blinking and constantly refreshing.

What can I do to fix this? Reading Vercel's documentation on this it states that the SSL certificate is automatically renewed prior to it expiring, yet mine have expired for both websites.

Can anyone offer any insight into how to fix this please?

Thanks

### Steps to Reproduce

Not really sure what to put here as its an account/domain issue.

**Comment by amyegan** [comment]
If you share the domain here, then I can take a look at the records to help you get this solved. 

If you prefer not to share the domain, here are a few guides should help: 
- [Common SSL certificate issues](https://vercel.com/docs/concepts/projects/domains/troubleshooting#common-ssl-certificate-issues)
- [Why is my domain not automatically generating an SSL certificate?](https://vercel.com/guides/domain-not-generating-ssl-certificate)
- [When is the SSL Certificate on my Vercel Domain renewed?](https://vercel.com/guides/renewal-of-ssl-certificates-with-a-vercel-domain)

**Reply by JonQuayle** [reply]
Thanks @amyegan. It seems to be happening on both domains which are **jonquayle.co.uk** and **beyondordinary.digital**.

**Reply by amyegan** [reply]
Thanks for sharing that. I see that there is an `_acme-challenge` record associated with both of these domains. Please either remove the CAA record or add a TXT record with the value `0 issue "letsencrypt.org"`. The cert should generate automatically after the updated records propagate.

**Reply by JonQuayle** [reply]
Thanks for the links @amyegan , so my domain registrar cannot add CAA records - they don't support them (which I find odd). I have added `0 issue "letsencrypt.org"` as a TXT record, left it overnight and it hasn't fixed the problem.

Is there anything else to try to get the certificates to generate? Otherwise I'm thinking I might need to look at transferring these domains to someone who supports all records.
